TYPO3.Media/Documentation/DeveloperInformation/index.rst

```diff
@@ -0,0 +1,44 @@
+Developer Information
+=====================
+
+Extend Asset Validation
+-----------------------
+
+Imagine you need to extend the validation of assets. For example to prevent
+duplicate file names or to run copyright checks on images. You can now do so
```

Comment:
I'd leave out the "now", since that only makes sense right now and to those who know it's a new feature. Once merged it loses relevance…